packages/code-studio/src/embed-grid/src/App.scss

```diff
@@ -1,5 +1,5 @@
 // Import Deephaven styling so we can use it's variables
-@import '~@deephaven/components/scss/custom_webpack.scss';
+@import '@deephaven/components/scss/custom.scss';
```

Comment:
You didn't delete `custom_webpack.scss`, I guess we still need it for other consumers of `@deephaven/components`?

Reply:
Nope looks like it's not needed any more. I think it was needed before because of CRA and the way its webpack config handled relative paths/imports